app/controllers/application_controller.rb

```diff
@@ -1,2 +1,12 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  def after_sign_out_path_for(_scope)
+    root_path
+  end
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: %i[name email password])
+    # devise_parameter_sanitizer.permit(:account_update, keys: %i[name email password photo bio])
```

Comment:
- [ Required ] I think it would be better if you remove commented code from your code base, commented code makes your repository look unprofessional. Please rectify this.